zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java

```diff
@@ -276,19 +276,7 @@ public void setReturn(InterpreterResult value, Throwable t) {
 
   @Override
   public Object clone() throws CloneNotSupportedException {
-    Paragraph paraClone = (Paragraph) super.clone();
-    Map<String, Object> config = new HashMap<>(this.getConfig());
-    // Show the editor by default
-    String hideEditorKey = "editorHide";
```

Comment:
@felixcheung I forgot to mention that this part is removed on purpose. Showing all the editor by default will break original notebook's layout. For me it makes more sense to also copy the hide/show editor configuration. What do you think?